scripts/measure_baseline.py

```
@@ -0,0 +1,198 @@
+    if args.json:
+        metadata = {"command": [args.bin, "--bench", *args.filter, *args.bench_args],
+                    "cpus": args.cpus, "requested_rounds": args.rounds,
+                    "platform": platform.platform(), "cwd": os.getcwd(),
+                    "binary_sha256": hashlib.sha256(Path(args.bin).read_bytes()).hexdigest()}
+        revision = subprocess.run(["git", "rev-parse", "HEAD"], capture_output=True, text=True)
+        metadata["revision"] = revision.stdout.strip() if revision.returncode == 0 else None
+        metadata["git_status"] = subprocess.run(
+            ["git", "status", "--short"], capture_output=True, text=True).stdout
+        print(json.dumps({"metadata": metadata, "benchmarks": agg}, indent=2))
+    else:
+        print(f"| Benchmark | Rounds | Median (of medians) | Min Median | Max Median | Spread | Best Fastest |")
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)